Cargo.toml

```diff
@@ -20,8 +20,8 @@ unstable = [
 
 [dependencies]
 matches = "0.1.2"
-html5ever = "0.4"
-hyper = {version = "0.7", optional = true}
+html5ever = ">= 0.4, <=0.5"
```

Comment:
Do you know if `<=0.5` the same as `<=0.5.0` or `<0.6.0` ? I think we want the latter, since the former excludes e.g. 0.5.1.